app/src/main/res/layout/fragment_splash.xml

```diff
@@ -1,60 +1,70 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Pavlo's pulled back from using scroll views, and I think we should stop where we can. I feel like the Enable Exposures (Onboarding Finish) fragment is the only place that needs it.

Reply:
I think it's okay to use it in these cases. Otherwise users won't be able to see the whole screen on small devices, considering how big our `.svgs` are

Reply:
^ I think for some of these it is, but the splash screen especially should be a dynamic 1 pager. It's lower pri though, another task for an after-2.0-flow-is-done PR

Reply:
The svgs could be scaled.